build.sbt

```diff
@@ -20,6 +20,19 @@ lazy val friendImpl = project("friend-impl")
       lagomJavadslTestKit
     )
   )
+  .settings(
```

Comment:
These settings should be automatically set by `sbt-lagom-bundle`. It shouldn't be necessary to set any kind of settings for a Lagom service.